### PR TITLE
[Flow EVM] update stdlib balance to use UInt instead of UFix64 

### LINKED ISSUE #4961: [Flow EVM] adding `toAttoFlow` method to `EVM.Balance`
We need to add `toAttoFlow` function to `EVM.Balance` as described in the flip.

### PR DESCRIPTION
**Pull request by ramtinms**
closes #4961

### CODE REVIEW

**Comment by turbolent** [comment]
Maybe consider using `UInt` if negative balances should be prevented automatically

File: fvm/evm/stdlib/contract.cdc [hunk 1]
```diff
@@ -22,28 +22,47 @@ contract EVM {
...
     access(all)
     struct Balance {
 
-        /// The balance in FLOW
+        /// The balance in atto-FLOW
+        /// Atto-FLOW is the smallest denomination of FLOW (1e18 FLOW)
+        /// that is used to store account balances inside EVM 
+        /// similar to the way WEI is used to store ETH divisible to 18 decimal places.
+        access(all)
+        var attoflow: UInt
```
Comment (sideninja) [inline fvm/evm/stdlib/contract.cdc:37]: Just checking if you intentionally name it `atttoflow` not `attoFLOW` because it's a new name for metric?

**Review by sideninja — APPROVED** [review]

File: fvm/evm/evm_test.go [hunk 2]
```diff
@@ -199,7 +198,9 @@ func TestBridgedAccountWithdraw(t *testing.T) {
                                    let bridgedAccount <- EVM.createBridgedAccount()
                                    bridgedAccount.deposit(from: <-vault)
 
-                                   let vault2 <- bridgedAccount.withdraw(balance: EVM.Balance(flow: 1.23))
+								   let bal = EVM.Balance(0)
+								   bal.setFLOW(flow: 1.23)
+                                   let vault2 <- bridgedAccount.withdraw(balance: bal)
```
Comment (janezpodhostnik) [inline fvm/evm/evm_test.go:203]: minor: some mixing of tabs and spaces.

**Review by janezpodhostnik — APPROVED** [review]

File: fvm/evm/evm_test.go (hunk 2 above)
Comment (sideninja) [reply fvm/evm/evm_test.go:203]: sorry I have to https://www.youtube.com/watch?v=SsoOG6ZeyUI 
:D

Comment (ramtinms) [reply fvm/evm/evm_test.go:203]: fixed

**Pull request merged**